packit_service/worker/handlers/gitlab_handlers.py

```diff
@@ -0,0 +1,81 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+This file defines classes for job handlers specific for Gitlab hooks
+"""
+import logging
+
+from packit.config import JobType
+from packit_service.constants import PERMISSIONS_ERROR_WRITE_OR_ADMIN
+from packit_service.service.events import (
+    MergeRequestCommentGitlabEvent,
+    TheJobTriggerType,
+)
+from packit_service.worker.build import CoprBuildJobHelper
+from packit_service.worker.handlers import CommentActionHandler
+from packit_service.worker.handlers.abstract import (
+    required_by,
+    use_for,
+)
+from packit_service.worker.handlers.comment_action_handler import (
+    add_to_comment_action_mapping,
+    add_to_comment_action_mapping_with_name,
+    CommentAction,
+)
+from packit_service.worker.result import HandlerResults
+
+logger = logging.getLogger(__name__)
+
+
+@add_to_comment_action_mapping
+@add_to_comment_action_mapping_with_name(name=CommentAction.build)
+@use_for(JobType.build)
+@use_for(JobType.copr_build)
+@required_by(JobType.tests)
+class GitlabMergeRequestCommentCoprBuildHandler(CommentActionHandler):
+    """ Handler for MR comment `/packit copr-build` """
```

Comment:
Is this really needed? Cannot we reuse Github one?

Reply:
Yes, we can I made some mistake with git. This as not supposed to be here.